test/integration/azure/TESTING_AGAINST_AZURE.md

```diff
@@ -0,0 +1,44 @@
+# Testing Against Azure - Integration Testing
+
+## Problem Statement
+
+We want to test Azure-related InSpec resource against Azure itself. This requires a test fixture in Azure to exmine using InSpec.
+
+## General Approach
+
+We have a Terraform plan to setup and destroy test fixtures in Azure. When the environment is running we have a set of integration tests that may run against it.
+
+Credentials are handled via a `~/.azure/credentials` file.
```

Comment:
This is a bit mysterious. Can we offer readers a clue about where/how to get the info that goes in there? What it looks like?

Reply:
Yeah I'll do that and give more information on setting up the account you'll need.

Reply:
Yeah I'll do that and give more information on setting up the account you'll need.